src/pages/gallery.js

```diff
@@ -1,36 +1,67 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styled from "styled-components";
 import ReactDOM from "react-dom";
 import { withRouteData } from "react-static";
 import { Link } from "react-router-dom";
 import * as Victory from "victory";
 
 import Page from "../partials/page";
-import Icon from "../partials/icon";
 import Preview from "../partials/gallery/preview";
 
-class Gallery extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.scope = {
-      ...Victory,
-      _: require("lodash"),
-      React,
-      ReactDOM,
-      PropTypes
-    };
+const GalleryWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.2rem;
+
+  > * + * {
+    margin-top: 4rem;
+  }
+
+  @media ${({ theme }) => theme.mediaQuery.sm} {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-auto-rows: auto;
+    grid-gap: 3.6rem;
+
+    > * + * {
+      margin-top: 0;
+    }
   }
+`;
+
+const Title = styled.div`
```

Comment:
keep this as a `p` tag since it's holding text or h2s so that it's able to be seen in the document outline since technically these are subsections of the 'victory gallery'

Reply:
Sounds good, great catch!